Examples/Algorithms/TrackFittingChi2/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+add_library(
+  ActsExamplesTrackFittingChi2 SHARED
+  src/TrackFittingChi2Options.cpp
+  src/TrackFittingChi2Algorithm.cpp
+  src/TrackFittingChi2AlgorithmFunction.cpp)
+target_include_directories(
+  ActsExamplesTrackFittingChi2
+  PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>)
+target_link_libraries(
+  ActsExamplesTrackFittingChi2
+  PUBLIC
+    ActsCore
+    ActsExamplesFramework ActsExamplesMagneticField
+    Boost::program_options)
```

Comment:
after you refactor @benjaminhuth we should not directly link to boost options right?

Reply:
yeah would be cool if you could move the option files to `Examples/Run/Common...` (cpp to `src`, hpp to `include/ActsExamples/Options` I think)